vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -734,6 +743,34 @@ public Date getSubscriptionExpiration() {
     }
 
     /**
+     * @return The expiration date for the SVOD subscription, or null if the Video is not an SVOD video
+     * or if the user is not subscribed to SVOD
+     */
+    @Nullable
+    public Date getSvodExpiration() {
+        if (isSvod()) {
+            // isSubscription will validate and prevent possible npes
+            assert mMetadata.getInteractions().getSvod() != null;
+            return mMetadata.getInteractions().getSvod().getExpiration();
+        }
+        return null;
```

Comment:
The underlying implementation of `isSvod` is subject to change, so using it as a null check here is incorrect.

This method could be simplified to:

```java
SvodInteraction svodInteraction = getSvodInteraction();
return svodInteraction != null ? svodInteraction.getExpiration() : null;
```